test/Elastic.Apm.Feature.Tests/Features/api_key.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Api Key
```

Comment:
Ideally long term it'd be nice if these could be shared across agents. But for now we don't really have a solution for that.

Reply:
There is a shell script that just does this within python agent: https://raw.githubusercontent.com/beniwohli/apm-agent-python/a393b5f0e522ccbeef9a5d79e1efcc181f2180cc/tests/scripts/download_gherkin_features.sh

I plan do use something similar for Java agent implementation

Reply:
For the record, we are in the process of automating this for all the agents :)